hack/build/run-functional-tests.sh

```diff
@@ -37,4 +37,9 @@ arg_oc="${KUBECTL:+-oc-path=$KUBECTL}"
 
 test_args="${test_args} -ginkgo.v ${arg_master} ${arg_namespace} ${arg_kubeconfig} ${arg_kubectl} ${arg_oc}"
 
+echo 'Wait until all CDI Pods are ready'
+until [[ $(./cluster/kubectl.sh get deployment cdi-file-host -n kube-system -o jsonpath={.status.availableReplicas}) -eq 1 ]] && [[ $(./cluster/kubectl.sh get deployment cdi-deployment -n kube-system -o jsonpath={.status.availableReplicas}) -eq 1 ]]; do
```

Comment:
kube-system should be replaced with the namespace var, right?

Reply:
Yes, yes they should. I am fixing.